### DISCUSSION TITLE
Schmea not found

### DISCUSSION THREAD

**Discussion by jamontesg** [webonyx/graphql-php#1052] [Q&A] [answered]
Hi
none of examples  works for me.

 curl -d '{"query": "query { echo(message: \"Hello World\") }" }' -H "Content-Type: application/json" http://localhost:8080
{"error":{"message":"Class 'GraphQL\\Type\\Schema' not found"}}

there are aditional documentation for this examples ?

regards

**Comment by spawnia** [comment]
composer install

**Reply by jamontesg** [reply]
Hi thanks for your time Benedikt

I already have installed webonyx

this is my package installation.

admin@ip-10-10-0-200:~/scripts2/soap$ composer show -i
You are using the deprecated option "installed". Only installed packages are shown by default now. The --all option can be used to show all packages.
aws/aws-sdk-php 3.33.4 AWS SDK for PHP - Use Amazon Web Services in your PHP project
graham-campbell/result-type v1.0.4 An Implementation Of The Result Type
guzzlehttp/guzzle 6.5.5 Guzzle is a PHP HTTP client library
guzzlehttp/promises 1.5.1 Guzzle promises library
guzzlehttp/psr7 1.8.3 PSR-7 message implementation that also provides common utility methods
mtdowling/jmespath.php 2.6.1 Declaratively specify how to extract elements from a JSON document
phpoption/phpoption 1.8.1 Option Type for PHP
psr/http-message 1.0.1 Common interface for HTTP messages
ralouphie/getallheaders 3.0.3 A polyfill for getallheaders.
symfony/polyfill-ctype v1.23.0 Symfony polyfill for ctype functions
symfony/polyfill-intl-idn v1.24.0 Symfony polyfill for intl's idn_to_ascii and idn_to_utf8 functions
symfony/polyfill-intl-normalizer v1.24.0 Symfony polyfill for intl's Normalizer class and related functions
symfony/polyfill-mbstring v1.24.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php72 v1.24.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP vers...
symfony/polyfill-php80 v1.23.1 Symfony polyfill backporting some PHP 8.0+ features to lower PHP vers...
vlucas/phpdotenv v5.4.0 Loads environment variables from .env to getenv(), $_ENV and `$...
webonyx/graphql-php v0.7.2 A PHP port of GraphQL reference implementation

Webonyx installed with version 0.7.2

I am using PHP 7.4.25

Regards

**Reply by spawnia** [reply]
`0.7` is ancient - we are on `14.x` now.

**Comment by jamontesg** [comment, marked as answer]
Great
was my old  installed version
Thanks
